app/services/populate_database.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+class PopulateDatabase
+  def initialize(register)
+    @registers_client ||= RegistersClient::RegisterClientManager.new(cache_duration: 60)
```

Comment:
This will always be a new `RegisterClientManager` because `PopulateRegisterDataInDbJob` creates a new `PopulateDatabase` instance when the `perform` method is called.